src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -26,27 +26,38 @@ import { WORKSPACE_CREATE_APP_ID } from '../../../common/constants';
 
 import { cleanWorkspaceId } from '../../../../../core/public';
 import { DeleteWorkspaceModal } from '../delete_workspace_modal';
+import { getSelectedFeatureQuantities } from '../../utils';
 
-const WORKSPACE_LIST_PAGE_DESCRIPTIOIN = i18n.translate('workspace.list.description', {
+export interface WorkspaceListProps {
+  workspaceConfigurableApps$?: BehaviorSubject<PublicAppInfo[]>;
+}
+
+const WORKSPACE_LIST_PAGE_DESCRIPTION = i18n.translate('workspace.list.description', {
   defaultMessage:
     'Workspace allow you to save and organize library items, such as index patterns, visualizations, dashboards, saved searches, and share them with other OpenSearch Dashboards users. You can control which features are visible in each workspace, and which users and groups have read and write access to the library items in the workspace.',
 });
 
-export const WorkspaceList = () => {
+const emptyWorkspaceList: WorkspaceObject[] = [];
+
+export const WorkspaceList = (props: WorkspaceListProps) => {
   const {
     services: { workspaces, application, http },
   } = useOpenSearchDashboards();
 
   const initialSortField = 'name';
   const initialSortDirection = 'asc';
-  const workspaceList = useObservable(workspaces?.workspaceList$ ?? of([]), []);
+  const workspaceList = useObservable(
+    workspaces?.workspaceList$ ?? of(emptyWorkspaceList),
+    emptyWorkspaceList
+  );
   const [queryInput, setQueryInput] = useState<string>('');
   const [pagination, setPagination] = useState({
     pageIndex: 0,
     pageSize: 5,
     pageSizeOptions: [5, 10, 20],
   });
   const [deletedWorkspace, setDeletedWorkspace] = useState<WorkspaceAttribute | null>(null);
+  const configurableApps = useObservable(props.workspaceConfigurableApps$ ?? of(undefined));
```

Comment:
curious why don't we use `[]` as default value.

Reply:
This is a missed change, updated.